css/style.css

```diff
@@ -1 +1,126 @@
-/* here goes the css */
\ No newline at end of file
+/* here goes the navbar */
+
+/* layout */
+	/* reset css */
+	* {
+		padding: 0;
+		margin: 0;
+		list-style: none;
+		text-decoration: none;
+		border: 0;
+		color: #333;
+		box-sizing: border-box;
+		background-repeat: no-repeat;
+		
+	}
+
+	/* hero */
+	.container{
```

Comment:
[Optional] After wrapping all your content in the main tag add `margin: 0 auto;` to the container tag this will center the container.

Reply:
Done